### ISSUE TITLE
feBlend not properly rendered if applied to a group

### ISSUE THREAD

**Issue by tanty** [Kozea/CairoSVG#76]
Consider these 2 similar SVG images:
1. https://rawgit.com/tanty/design-hodgepodge/master/svg/filter/igalia.svg
2. https://rawgit.com/tanty/design-hodgepodge/master/svg/filter/igalia-feblend-no-grouped.svg

The only difference is that the feBlend filter is applied to a group of elements or the elements, individually. The PNG reference look for both should be:
https://raw.githubusercontent.com/tanty/design-hodgepodge/master/svg/filter/igalia.png

However, when using CairoSVG to export them to PDF, the results are:
1. https://rawgit.com/tanty/design-hodgepodge/master/svg/filter/igalia.pdf
2. https://rawgit.com/tanty/design-hodgepodge/master/svg/filter/igalia-feblend-no-grouped.pdf

In other words, when the feBlend filter is applied to a group of elements, it is not respected when exporting to PDF.

**Comment by liZe** [comment]
Thanks for reporting, it's fixed!

Your logo doesn't render correctly on firefox, chrome or epiphany, so I'm not the only one to have problems with SVG filters ;). By the way, I wouldn't seriously rely on CairoSVG for filters, even if this fix is not that ugly.
